docs/man/git-lfs-ls-files.1.ronn

```diff
@@ -16,6 +16,10 @@ An asterisk (*) after the OID indicates a LFS pointer, a minus (-) a full object
 * `-l` `--long`:
   Show the entire 64 character OID, instead of just first 10.
 
+* `-s` `--size`:
+  Show the size in bytes of the LFS object between parenthesis at the end of a
+  line.
```

Comment:
I very much like this option 👍 

However, I feel this is an option for humans and therefore the byte size might be bit odd. I think MB as a default would be much more readable. I do agree that printing bytes is "technically more correct", though. However, robots can use the `--debug` option to get the exact byte size, no?

Reply:
Great call -- I pushed up something I think is somewhere down the middle in 11f6834. It uses the `humanize` package to find the biggest unit of storage that preserves accuracy for a given size. `git-lfs-ls-files(1)` uses that to print the biggest unit of storage it can, and if users/parser want something more specific they can use `--debug`.

Reply:
That might print different units in a single call, right?
e.g.:
```
d1b2a59fbe * a.dat (8 B)
a1b2a59fbe * big.dat (7 MB)
c1b2a59fbe * a.dat (1 GB)
```
I wonder if this makes it harder to less hard to read for humans.
Sorry for the 🚲 -shedding 😄

Reply:
It could, but I'm not sure what a better alternative would be. It might be inaccurate to assume for all objects in all repositories that "MB" is the most accuracy-preserving unit. What do you think?

Reply:
After thinking about it I believe `humanize` is the best approach here 👍

Reply:
Nit in the docs:
```
s/in bytes//
```